src/workerd/io/trace.c++

```diff
@@ -126,51 +126,54 @@ kj::String TraceId::toW3C() const {
   return kj::str(s.releaseAsArray());
 }
 
-TraceId TraceId::fromEntropy(kj::Maybe<kj::EntropySource&> entropySource) {
-  if (isPredictableModeForTest()) {
-    return TraceId(0x2a2a2a2a2a2a2a2a, 0x2a2a2a2a2a2a2a2a);
-  }
-
-  uint64_t low = 0;
-  uint64_t high = 0;
+namespace {
+uint64_t getRandom64Bit(kj::Maybe<kj::EntropySource&>& entropySource) {
```

Comment:
```suggestion
// Generate a random 64-bit number. If an entropy source is given, use it,
// otherwise fall back to RAND_bytes. If, by some chance, the random
// generator returns 0, we'll try up to three times before just returning.
uint64_t getRandom64Bit(kj::Maybe<kj::EntropySource&>& entropySource) {
```